substrate/primitives/keyring/src/ed25519.rs

```diff
@@ -134,6 +134,39 @@ impl From<Keyring> for sp_runtime::MultiSigner {
 	}
 }
 
+#[derive(Debug)]
+pub struct ParseKeyringError;
```

Comment:
nit: could be shared between mods.

Reply:
> nit: could be shared between mods.

Move it to `lib.rs`?

Reply:
yeah, I would try that. @iulianbarbu wdyt?

Reply:
I would certainly reuse the error at the crate level and put the error in `lib.rs`, and then use it in `sr25519` module too.